guides/common/modules/proc_applying-errata-to-hosts.adoc

```diff
@@ -18,31 +21,36 @@ Navigate to *Hosts* > *Host Collections* and select a host collection.
 Go to *Collection Actions* > *Errata Installation* and notice the search query box contents.
 For example, for a Host Collection called _my-collection_, the search box contains `host_collection="my-collection"`.
 
-[id="exam-API_Guide-Applying_Errata_to_a_Host"]
+[id="api-appyling-errata-to-a-host"]
 .Applying errata to a host
```

Comment:
If we make this a "real" heading (in a separate file; level2 for demo purpose only):

```suggestion
== Applying errata to a host
```

Then we could change line 29 to

````diff
-Examaple request:
+.API request
````

We could then use `.API reponse` (if applicable); see https://github.com/theforeman/foreman-documentation/pull/3353/files#diff-272b3840cb6e38e03fe0ebbc24651faa6e195076e4fdd41ef368456fd3c9e28aL82

What do you think?

Reply:
I like that idea!

Reply:
Applied to **all** files :)